composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "scandipwa/compare-graphql",
+    "description": "N/A",
+    "version": "0.0.1",
+    "type": "magento2-module",
+    "require": {
+        "magento/product-community-edition": "^2.3.1"
+    },
+    "support": {
+        "source": "https://github.com/scandipwa/compare-graphql",
+        "email": "info@scandiweb.com"
+    },
+    "authors": [
+        {
+            "name": "Alfreds Genkins",
+            "email": "alfreds@scandiweb.com"
+        },
+        {
+            "name": "Lauris Grosu",
+            "email": "lauris.grosu@scandiweb.com"
+        }
```

Comment:
We are not authors anymore, you can safely remove us and put your name there :D

Reply:
Done.